tsconfig.json

```diff
@@ -45,6 +45,8 @@
     "frontend/src/pages/audio/_id/index.vue",
     "frontend/src/pages/image/_id/index.vue",
     "frontend/src/pages/image/_id/report.vue",
+    "frontend/src/pages/search.vue",
+    "frontend/src/pages/search/**/*.vue",
```

Comment:
Awesome! By my count, there are only three pages left to add:

- preferences
- privacy
- sources

Should we open issues for that? They all seem like potential candidates for "help wanted" labels :thinking:

Reply:
I opened an issue for that: #2865 . Do you think it should be split into separate issues and "enriched" to be a better "help wanted" issues?

When I added all of the pages to `tsconfig`, there were no errors in the `sources` page, but `index` page did have errors.

Reply:
Up to you! It sounds like it might be simple enough to do in one PR, in which case it's overcomplicating things to put it into different ones for no reason.